liblangutil/ParserBase.cpp

```diff
@@ -85,6 +85,45 @@ void ParserBase::expectToken(Token _value, bool _advance)
 		m_scanner->next();
 }
 
+void ParserBase::expectTokenRecoveryDelete(Token _value, bool _advance)
+{
+	Token tok = m_scanner->currentToken();
+	if (tok != _value)
+	{
+		auto tokenName = [this](Token _token)
+		{
+			if (_token == Token::Identifier)
+				return string("identifier");
```

Comment:
We didn't actually clarify that in our coding guidelines yet, but lately we have some occurrences of curly braced initializers/constructors instead of function-style constructors. But with the change above, you can skip that (and the 2 spots below) completely and just return the literal.

Reply:
I made no change here. As with some of the other changes you are basically suggesting that the existing code get altered in way that I do not feel i am proficient at. 

So as with the other things, it would be better if you suggested coding changes or create an issue for this. Thanks.